src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -452,123 +454,162 @@ struct GhValenciaDivCleanTemplateBase<
       volume_dim, analytic_solution_fields, use_dg_subcell>;
   using error_compute = Tags::ErrorsCompute<analytic_solution_fields>;
   using error_tags = db::wrap_tags_in<Tags::Error, analytic_solution_fields>;
-  using observe_fields = tmpl::push_back<
-      tmpl::append<
-          typename system::variables_tag::tags_list,
-          typename system::primitive_variables_tag::tags_list,
-          tmpl::conditional_t<use_numeric_initial_data, tmpl::list<>,
-                              error_tags>,
-          tmpl::conditional_t<
```

Comment:
why is the conditional no longer needed?

Reply:
We can compute the quantities with or without the control system present, so it seem a bit silly to have all the nested, indentation, etc. just for that.